addons/ekco/0.9.0/install.sh

```diff
@@ -0,0 +1,234 @@
+
+function ekco_pre_init() {
+    if [ -z "$EKCO_NODE_UNREACHABLE_TOLERATION_DURATION" ]; then
+        EKCO_NODE_UNREACHABLE_TOLERATION_DURATION=5m
+    fi
+    if [ -z "$EKCO_MIN_READY_MASTER_NODE_COUNT" ]; then
+        EKCO_MIN_READY_MASTER_NODE_COUNT=2
+    fi
+    if [ -z "$EKCO_MIN_READY_WORKER_NODE_COUNT" ]; then
+        EKCO_MIN_READY_WORKER_NODE_COUNT=0
+    fi
+    EKCO_SHOULD_MAINTAIN_ROOK_STORAGE_NODES=true
+    if [ "$EKCO_ROOK_SHOULD_USE_ALL_NODES" = "1" ]; then
+        EKCO_SHOULD_MAINTAIN_ROOK_STORAGE_NODES=false
+    fi
+    EKCO_SHOULD_INSTALL_REBOOT_SERVICE=1
+    if [ "$EKCO_SHOULD_DISABLE_REBOOT_SERVICE" = "1" ]; then
+        EKCO_SHOULD_INSTALL_REBOOT_SERVICE=0
+    fi
+    EKCO_PURGE_DEAD_NODES=false
+    if [ "$EKCO_SHOULD_ENABLE_PURGE_NODES" = "1" ]; then
+        EKCO_PURGE_DEAD_NODES=true
+    fi
+    EKCO_CLEAR_DEAD_NODES=true
+    if [ "$EKCO_SHOULD_DISABLE_CLEAR_NODES" = "1" ]; then
+        EKCO_CLEAR_DEAD_NODES=false
+    fi
+    if [ "$ROOK_VERSION" = "1.0.4" ]; then
```

Comment:
This block is new

Reply:
is there a way for someone to turn off rook priority classes?

Reply:
You can manually edit the ekco configmap in the kurl namespace and remove the rook-priority-class option.